.github/mapchecker/whitelist.yml

```diff
@@ -0,0 +1,46 @@
+Frontier:
```

Comment:
for some of these, we might want the whole POI to just be skipped, like the frontier outpost or the upcoming nfsd poi

Reply:
Can do as well, but the intention for this is for these kinds of changes to stand out. It's not foolproof, something that was already whitelisted can be added infinitely.

If someone wants to add a restricted prototype to their ship, I want them to have to edit the whitelist file to make it stand out that they are requesting that exception from regular mapping rules. Adding this based on the check output is pretty easy, as it tells you exactly which map name needs to have exactly which prototype names addressed.